### PR TITLE
Support for YUV420 format.

### PR DESCRIPTION
**Pull request by smimram**

### CODE REVIEW

**Comment by smimram** [comment]
@toots I think that this can be merged, can I go ahead?

This is mostly backward compatible and prepares the transition for YUV420. The only required changes in Liquidsoap are https://github.com/savonet/liquidsoap/pull/854

File: CHANGES [hunk 1]
```diff
@@ -1,9 +1,14 @@
 0.5.0 (unreleased)
-=====
-* Fix memory leak in `RGBA32.of_RGB24_string`.
+=======
 * New implementation of `YUV420`, added many function to manipulate those.
 * Enhanced `Video` module.
 
+0.4.1 (27-06-2019)
+* Fix memory leak in `RGBA32.of_RGB24_string`.
+=====
```
Comment (toots) [inline CHANGES]: Something's off here 😅

File: src/image_rgb.c [hunk 2]
```diff
@@ -160,8 +119,11 @@ static frame *rgb_copy(frame *src, frame *dst)
   dst->width = src->width;
   dst->height = src->height;
   dst->stride = src->stride;
-  dst->data = memalign(ALIGNMENT_BYTES, Rgb_data_size(src));
+  dst->data = malloc(Rgb_data_size(src));
   if (dst->data == NULL) caml_raise_out_of_memory();
+  // Disabling this since we are forgetting about dst->data without freeing:
+  // possible memory leak...
+  assert(0);
```
Comment (toots) [inline src/image_rgb.c]: ?

File: src/image_rgb.c [hunk 3]
```diff
@@ -160,8 +119,11 @@ static frame *rgb_copy(frame *src, frame *dst)
   dst->width = src->width;
   dst->height = src->height;
   dst->stride = src->stride;
-  dst->data = memalign(ALIGNMENT_BYTES, Rgb_data_size(src));
```
Comment (toots) [inline src/image_rgb.c]: No more aligned memory?

File: src/image_rgb.c (hunk 2 above)
Comment (smimram) [reply src/image_rgb.c]: I was wrong, there is no problem here, removing the assert.

File: src/image_rgb.c (hunk 3 above)
Comment (smimram) [reply src/image_rgb.c]: Was it really useful? The constant `ALIGNMENT_BYTES` seemed pretty arbitrary to me...

**Pull request merged**